packages/wrap/src/transforms/FilterRootFields.ts

```diff
@@ -6,8 +6,12 @@ import { SubschemaConfig, Transform } from '@graphql-tools/delegate';
 
 import TransformRootFields from './TransformRootFields';
 
-export default class FilterRootFields implements Transform {
-  private readonly transformer: TransformRootFields;
+interface FilterRootFieldsTransformationContext extends Record<string, any> {}
```

Comment:
If you don't think this named interface / alias for `Record<string, any>` thing I did in all the files I will get rid of it. Maybe it doesn't make sense or is not even correct.